backend/src/main/resources/application-dev.yml

```
@@ -0,0 +1,18 @@
+      hibernate:
+        format_sql: true
+
+logging:
+  level:
+    org.hibernate.SQL: DEBUG # SQL logs uniquement en dev
+    org.hibernate.type.descriptor.sql.BasicBinder: TRACE # paramètres SQL en dev
+
+management:
+  endpoint:
+    health:
+      show-details: always # en dev, exposer les détails pour le diagnostic
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)